src/runtimes/wapc.rs

```diff
@@ -65,7 +65,7 @@ pub(crate) fn host_callback(
 
                     send_request_and_wait_for_response(policy_id, binding, operation, req, rx)
                 }
-                "v1/manifest_digest" => {
+                "v2/manifest_digest" => {
```

Comment:
why we need to bump to v2 here? Isn't it enough just in verify?

Reply:
true! fixed in https://github.com/kubewarden/policy-evaluator/pull/152/commits/ae70eaf5a84e46fa9747a2a65f7619547307845c, rebased against sdk 0.6.2.